earthpy/tests/test_crop_image.py

```diff
@@ -37,6 +39,21 @@ def basic_geometry_gdf(basic_geometry):
     return gdf
 
 
+@pytest.fixture
+def output_dir(out_path):
+    return out_path[:-8]
+
+
+@pytest.fixture
+def output_file_list(output_dir):
+    output_file_list = []
+    for i in range(4):
+        output_file_list.append(
+            os.path.join(output_dir, "file" + str(i) + ".tif")
+        )
+    return output_file_list
+
+
 def test_crop_image_with_gdf(basic_image_tif, basic_geometry_gdf):
```

Comment:
@nkorinek for every other function we have a separate file for tests. i see why you combined these but i suggest that you divide crop_all out into a new file so we are consistent.